### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_SchemaCompareOptions_SchemaCompare_Screenreader: When navigated to checkbox, nvda doesn't announce as 'checkbox' along with it's status.

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6726]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 975e45061b073ecce2398d00c12f525d90e07b02
Date: 2019-08-08T19:22:13.572Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Nvda Version: 2019.2rc2

**Additional Details:**
Tools Name: Nvda
MAS References: MAS1.3.1

**Repro Steps:**

1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Options menu from Schema compare Menu bar & hit enter.
12. Start screen reader.
13. Navigate to the checkbox of the first row and verify and then to any checked/ unchecked checkbox.

**Actual:**
Nvda announces as checkbox with its status only when navigated to row for the first time. When again navigated, Nvda announces as 'Selected' for checked and unchecked Checkboxes .

**Expected:**
When navigated to any checkbox, NVDA should announce control type as checkbox along with it's status as 'Checked'/ 'Unchecked'.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader users will not know that focus is moved to checkbox and so will be unable to perform any action on it.

**MAS Reference:**
MAS 1.3.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
![image](https://user-images.githubusercontent.com/53556874/62934975-8bab5c80-bde3-11e9-9cad-99dab4cdf8e9.png)

Observations from the image:
Nvda announces as 'Selected' when navigated to checked Checkbox present in 2nd row and 3rd row.
